great_expectations/data_context/store/html_site_store.py

```diff
@@ -213,6 +214,9 @@ def write_index_page(self, page):
     def clean_site(self):
         for _, target_store_backend in self.store_backends.items():
             keys = target_store_backend.list_keys()
+            #if no keys are found, still remove default site
```

Comment:
I don't follow this logic -- what's the issue with leaving it alone?

Reply:
If user came with --all and didn't create a new site, clean presumably means clean default site.